src/groundlight/client.py

```diff
@@ -72,6 +72,15 @@ def __init__(self, endpoint: Optional[str] = None, api_token: Optional[str] = No
         self.detectors_api = DetectorsApi(self.api_client)
         self.image_queries_api = ImageQueriesApi(self.api_client)
 
+    @classmethod
+    def _post_process_image_query(cls, iq: ImageQuery) -> ImageQuery:
+        """Post-process the image query so we don't use confusing internal labels.
+
+        TODO: Get rid of this once we clean up the mapping logic server-side.
+        """
+        iq.result.label = convert_internal_label_to_display(iq, iq.result.label)
+        return iq
```

Comment:
Here are some alternatives to consider:
- we could put this conversion (and the `is_valid` validators) directly in the Pydantic `ImageQuery` model. The caveat is that `ImageQuery` and `ClassificationResult` are auto-generated from the OpenAPI spec, so we could accidentally lose the validator on next auto-generation.
- we could put this conversion (and the `is_valid` validators) in a light SDK Pydantic model wrapper or subclass. That way, we know it won't be overwritten by code geenration. But then we introduce another source of differences between the API and the SDK -- which can be good and bad.

Reply:
Either way it's a difference between the API & SDK, right?

I agree with not messing with the auto-generated code.  So putting it in a pydantic model seems reasonable.  But I'm also fine with this for now.